tox.ini

```diff
@@ -19,12 +19,8 @@ passenv = *
 
 commands =
     /usr/bin/find "{toxinidir}" -name '*.pyc' -delete
-    coverage run {env:COVERAGE_OPTS:} --include=s3_storage_provider.py \
-        "{envbindir}/trial" {env:TRIAL_FLAGS:} {posargs:test_s3} {env:TOXSUFFIX:}
-    {env:DUMP_COVERAGE_COMMAND:coverage report -m}
```

Comment:
Given we're dropping these environment variables, can we also remove the `PEP8SUFFIX` here? https://github.com/matrix-org/synapse-s3-storage-provider/blob/236e0cddb8e176debccd77321a5d6ed6d44dc617/tox.ini#L53